src/status_im/ui/screens/offline_messaging_settings/events.cljs

```diff
@@ -0,0 +1,11 @@
+(ns status-im.ui.screens.offline-messaging-settings.events
+  (:require [status-im.utils.handlers :as handlers]
+            [status-im.ui.screens.accounts.events :as accounts-events]))
+
+(handlers/register-handler-fx
+  :connect-wnode
+  (fn [{:keys [db now]} [_ wnode]]
+    (merge (accounts-events/account-update {:db db} {:wnode        wnode
```

Comment:
@oskarth better now?
```diff
-    (merge (accounts-events/account-update {:db db} {:wnode        wnode
-                                                     :last-updated now})
-           {:dispatch     [:navigate-to-clean :accounts]
-            :stop-whisper nil})))
+    (as-> {:wnode        wnode
+           :last-updated now} $
+        (accounts-events/account-update {:db db} $)
+        (assoc $ :dispatch     [:navigate-to-clean :accounts]
+                 :stop-whisper nil))))
```

Reply:
Sorry, not really :P I'd say the first was better. Why not something like this?

```
(-> (accounts-events/account-update {:db db}
                                    {:wnode wnode :last-updated now})
     (merge {:dispatch    [:navigate-to-clean :accounts]
            :stop-whisper nil}))
```
Or this?
```
(let [acc-fx (accounts-events/account-update {:db db}
                                             {:wnode wnode :last-updated now})]
  (merge acc-fx 
         {:dispatch    [:navigate-to-clean :accounts]
         :stop-whisper nil}))
```

Or even put other fx in let too and merge at end.

Since this is quite a lot of effects IMO, maybe we could spec as a post condition what we expect the shape of the data to look like? At least loosely.